hibernate-spatial/src/test/java/org/hibernate/spatial/testing/TestDataReader.java

```diff
@@ -26,12 +26,18 @@
 		List<TestDataElement> testDataElements = new ArrayList<TestDataElement>();
 		SAXReader reader = new SAXReader();
 		try {
+			reader.setFeature("http://apache.org/xml/features/nonvalidating/load-external-dtd", false);
+			reader.setFeature("http://xml.org/sax/features/external-general-entities", false);
+			reader.setFeature("http://xml.org/sax/features/external-parameter-entities", false);
 			Document document = reader.read( getInputStream( fileName ) );
 			addDataElements( document, testDataElements );
 		}
 		catch (DocumentException e) {
 			throw new RuntimeException( e );
 		}
```

Comment:
Could we remove the above catch case for the below one makes it unnecessary?

Reply:
right, thanks @NathanQingyangXu .

@gbadner : don't worry I'll apply Nathan's suggestion, adapt the code style (whitespace formatting) and merge it already. Thanks!